packages/manager-components/src/components/filters/pci/filter-add.component.tsx

```diff
@@ -27,8 +28,8 @@ export interface FilterAddProps {
   onAddFilter: (filter: Filter, column: ColumnFilter) => void;
 }
 
-export default function FilterAdd({ columns, onAddFilter }: FilterAddProps) {
-  const { t } = useTranslation('filter');
+export function FilterAdd({ columns, onAddFilter }: FilterAddProps) {
+  const { t } = useTranslation('pci-datagrid-filters');
```

Comment:
i don't think it should be scoped to "pci" since the filters are generic it could be used in any universe

Reply:
Fixed